src/logic/operate.js

```diff
@@ -0,0 +1,26 @@
+import Big from 'big.js';
+
+export default function operate(numberOne, numberTwo, operation) {
+  const one = Big(numberOne);
+  const two = Big(numberTwo);
+  if (operation === '+') {
+    return one.plus(two).toString();
+  }
+  if (operation === '-') {
+    return one.minus(two).toString();
+  }
+  if (operation === 'x') {
+    return one.times(two).toString();
+  }
+  if (operation === '÷') {
+    try {
+      return one.div(two).toString();
+    } catch (err) {
+      return "Can't divide by 0.";
+    }
+  }
+  if (operation === '%') {
+    return one.mod(two).toString();
+  }
```

Comment:
- [ ] Your app crashes when I try to find modulus using `0`. This is because your `operate.js` file is missing the right **try** and **catch** code for the `%` operation. Kindly use the provided file. You are required to copy this file as provided in the curriculum: [operate.js file](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/code-samples/operate.js)

  ![bright-modulo](https://user-images.githubusercontent.com/36542986/152951289-8d482ab8-c9d7-46e0-b24d-be50b4d7a90e.jpg)

Reply:
Done!